test/bn_rand_range.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * WARNING: do not edit!
+ * Generated by test/bn_rand_range.py
```

Comment:
This script doesn't exist any more in the test dir. Perhaps there should be a comment referring to the tools repo

Reply:
Good point, I updated the script with this information but didn't regenerate the header file.  I'll do so.

Reply:
Done.